MODULE.bazel

```diff
@@ -68,6 +69,21 @@ use_repo(
     "rules_pkl_deps",
     "unpinned_rules_pkl_deps",
 )
+maven.install(
+    name = "custom_pkl_java_library_maven_deps",
```

Comment:
We're in a `bzlmod` world. It would be nice to move this somewhere else to make the root `MODULE.bazel` file neater 🤔

Reply:
Seems we can't use `include()` in our `MODULE.bazel` due to how we're using `bazel_integration_test`. It gets confused. Leaving this in place, but adding a comment.